main/src/modules/jointsAwareness/include/jointsAwareness.h

```diff
@@ -0,0 +1,88 @@
+#include <iostream>
+#include <sstream>
+#include <map>
+#include <fstream>
+#include <string>
+#include <map>
+#include <yarp/dev/Drivers.h>
+#include <yarp/dev/CartesianControl.h>
+#include <yarp/dev/GazeControl.h>
+#include <yarp/dev/PolyDriver.h>
+#include "wrdac/clients/icubClient.h"
+
+
+
+using namespace wysiwyd::wrdac;
```

Comment:
Hi @maxime-petit, please don't use `using namespace` in header files. It's evil!